ui/create_restore_page.go

```diff
@@ -0,0 +1,555 @@
+package ui
+
+import (
+	"fmt"
+	"image/color"
+	"strings"
+
+	"gioui.org/io/key"
+
+	"github.com/raedahgroup/dcrlibwallet"
+	"github.com/raedahgroup/godcr/wallet"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+	"github.com/raedahgroup/godcr/ui/decredmaterial"
+	"golang.org/x/exp/shiny/materialdesign/icons"
```

Comment:
On a lighter note, could we try rearranging the imports? standard imports at the top, other imports below?. I think it makes our work cleaner and more orderly.

Reply:
> On a lighter note, could we try rearranging the imports? standard imports at the top, other imports below?. I think it makes our work cleaner and more orderly.

doesn't really matter. As long as it builds on the CI, it's fine.